subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_fota

```diff
@@ -90,8 +90,8 @@ endif # NRF_CLOUD_REST
 
 menuconfig NRF_CLOUD_FOTA_FULL_MODEM_UPDATE
 	bool "Enable full modem FOTA updates"
+	depends on ZCBOR
```

Comment:
in the `nrf_cloud_multi_service`, `asset_tracker_v2`, and `nrf_cloud_rest_fota` samples, please add `CONFIG_ZCBOR=y` to the .conf files that set `CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE=y`.

Reply:
Related to my comment above, is there some condition we can check that is common for all of these?

Reply:
`NRF_CLOUD_FOTA_FULL_MODEM_UPDATE` and `NRF_CLOUD_COAP` would be enabled by the application, so I don't know know of any such condition.